02/app.js

```diff
@@ -1,2 +1,25 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for(let i = 1; i <= x; i++) {
+    if(i===x) {
+        result=result+i;
+        console.log(result);
+    } else {
+        result=result+i;
+    }
+}
```

Comment:
Nie za bardzo rozumiem koncepcję tego warunku. Zobacz, że zwiększanie wartości dla `result` masz równocześnie w `if` oraz w `else` więc można to uprościć:
```
for(let i = 1; i <= x; i++) {
     result=result+i;
    if(i===x) {
        console.log(result);
    }
}
```

Skoro chcemy wyświetlić result kiedy `x === i` to w zasadzie można zrobić to poza pętlą tj.
```
for(let i = 1; i <= x; i++) { // zwróć uwagę na warunek pętla działa do póki `i <= x` (i jest mniejsze lub równe x)
     result=result+i;
}

console.log(result);
```